File: 1-Iniciante/ex3303.c
#include <stdio.h>
#include <string.h> 

int main(){
    char palavra[20];
    scanf("%s", palavra);

    int tam = strlen(palavra);

    if(tam > 9){printf("palavrao\n");}
    else{printf("palavrinha\n");}
    
    return 0;
}

File: 1-Iniciante/Ex3303.java
import java.util.Scanner;

public class Ex3303 { // para ser aceito no URI mude o nome da classe para Main
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String palavra;

        palavra = scanner.nextLine();

        if (palavra.length() > 9) {
            System.out.println("palavrao");
        } else {
            System.out.println("palavrinha");
        }
        scanner.close();
    }
}